solution/index.js

```diff
@@ -0,0 +1,356 @@
+const sections = document.querySelectorAll('.section')
+const buttons = document.querySelectorAll('button')
+const uls = document.querySelectorAll('ul')
+const searchBar = document.querySelector('#search')
+
+if (!localStorage.tasks) {
+  localStorage.setItem('tasks', '{"todo": [],"in-progress": [],"done": []}') // makes sure there is always a tasks key in local storage
+}
+
+const parsedTasks = JSON.parse(localStorage.getItem('tasks'))
+
+function paintDomFromLocalStorage(dataObject) {
+  const neaterUlNames = []
+  uls.forEach((ul) => neaterUlNames.push(ul.className.split('-tasks')[0]))
+  // this leaves nicer names to work with
+  for (let key in dataObject) {
+    for (let i = 0; i < neaterUlNames.length; i++) {
+      if (key.split('-').join('') === neaterUlNames[i].split('-').join('')) {
+        // find the matching ul to the mathching key (todo & to-do)
+        for (let j = 0; j < dataObject[key].length; j++) {
+          // iterate over the property in the parsed version of the local storage and put it on the DOM
+          const newLi = createLiElement(dataObject[key][j], ['task'])
+          uls[i].append(newLi)
+        }
+      }
+    }
+  }
+}
+paintDomFromLocalStorage(parsedTasks)
+
+const liS = document.querySelectorAll('li')
+liS.forEach((li) => {
+  li.addEventListener('mouseenter', altHandlerFunction)
+})
+
+function altHandlerFunction(e) {
+  // make sure this only runs while cursor is on <li>
+  const liElements = document.querySelectorAll('li') // a different var than above var (for the initial use of the app, the app doesn't know liS until the first refresh)
+  liElements.forEach((li) =>
+    li.addEventListener('mouseleave', () => {
+      document.removeEventListener('keydown', addToObject)
+    })
+  )
+  const numberOptions = ['1', '2', '3']
+
+  function addToObject(altClickEvent) {
+    const keyPressedObject = {}
+    keyPressedObject[altClickEvent.key] = true
```

Comment:
The correct way to check if `alt` is pressed is via `event.altKey` (for keyboard events).
You could check if both `alt` and `1`/`2`/`3` are pressed in the same event